string_utils/fms_string_utils_binding.c

```diff
@@ -162,3 +162,20 @@ char* fms_find_my_string_binding(char** arr, int *n, char *find_me, int *np)
   strcpy(string_p, string);
   return string_p;
 }
+
+// Finds the number of unique strings in an array
+int fms_find_unique(char** arr, int *n)
```

Comment:
These C function variables should be documented https://www.doxygen.nl/manual/commands.html#cmdparam